to-do-list.js

```diff
@@ -2,15 +2,27 @@ const input = document.getElementById('text-field');
 
 const createListItem = function() {
   const listItem = document.createElement('li');
+  const listItemSpan = document.createElement('span');
+  listItemSpan.classList.add('uncompleted');
+  listItemSpan.addEventListener('click', function(event) {
```

Comment:
Extract to separated function, pass here by reference

Reply:
Ok. I struggled to do this as I had a problem with scope of variables or hoisting. Not sure. I don't event understand how hoisting works.
I somehow succeeded, however I am not sure if this is correct solution.